### DISCUSSION TITLE
Is there a way to stop rewriting the console content?

### DISCUSSION THREAD

**Discussion by oom-** [nuxt/nuxt#30124] [Questions]
👋🏻 I don't know why, but when I'm trying to debug and write `console.log` in my routes on the `server` side the logs keep getting erased instantly (I tried with `--no-clear` option but it's seems that's just for startup): 

![Code_2QXNM91XhR](https://github.com/user-attachments/assets/51ed6ced-bc25-4d86-aac7-6531f025ec9c)

```js
export default defineEventHandler({
    onRequest: [apiRouteAdminCheck],
    handler: async (event) => {
        //get body code
      
        console.log("<=== BODY RECEIVED: ", body);

        //Update code

        setResponseStatus(event, 204);
        return;
    }
});

```

**Comment by jwhazel** [comment]
Is there a chance you have any Nuxt plugins or middleware installed? We've run across something very similar and found it was due to a 3rd party plugin.

**Reply by oom-** [reply]
Thank you for your answer, I have a few of them, does one of these could match one you have?
-    "@nuxt/image": "^1.8.1"
-    "@pinia/nuxt": "^0.9.0"
-    "tailwindcss": "^3.4.16",

I have as well 4 custom middleware, 3 are using a store to perform check and 1 is adding meta to the route.
